extension/reearth-yml.ts

```diff
@@ -213,6 +213,60 @@ const yml = {
         },
       },
     },
+    {
+      id: "notification-widget",
+      type: "widget",
+      name: "Notification",
+      widgetLayout: {
+        extendable: {
+          horizontally: true,
+        },
+        defaultLocation: {
+          zone: "outer",
```

Comment:
```suggestion
          zone: "inner",
```
Locate it in `outer` will making Re:Earth Visualizer show two pages when in mobile layout.
Currently we only have `Editor` in `outer` but it's not included in the final project so that's okey.

Reply:
I can understand, thank you! 
and I could fix it;
![image](https://github.com/user-attachments/assets/7ce1ff1e-83b7-438d-b833-c3b414d33c96)